test/e2e/bootstrapper_test.go

```diff
@@ -2,23 +2,23 @@ package e2e
 
 import (
 	"fmt"
+	"github.com/openshift/windows-machine-config-operator/pkg/bootstrapper"
+	"github.com/spf13/pflag"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
 	"io/ioutil"
 	"os"
 	"strings"
 	"testing"
 	"time"
-
-	"github.com/openshift/windows-machine-config-operator/pkg/bootstrapper"
-	"github.com/spf13/pflag"
-	"github.com/stretchr/testify/assert"
-	"golang.org/x/sys/windows/svc"
-	"golang.org/x/sys/windows/svc/mgr"
```

Comment:
Formatting changes should have their own PR but I will let it go this time.

Reply:
I should have checked for the formatting before the commit - I will revert this.